### ISSUE TITLE
Does TechSpy still exists?

### ISSUE THREAD

**Issue by AsenOsen** [leapsecurity/InSpy#35]
Did not found anythong useful in version 3.0.1. Did I miss TechSpy routine?

**Comment by Alaponuke** [comment]
yes I have same problem neither techspy nor empspy are worked always said :
inspy: error: unrecognized arguments: --techspy
